test/transactions.go

```diff
@@ -12,6 +16,7 @@ func TransferTransaction() *transferTransaction {
 			Transaction: &protocol.TransactionBuilder{
 				ContractName: "BenchmarkToken",
 				MethodName:   "transfer",
+				Timestamp:    primitives.Timestamp(time.Now().Unix()),
```

Comment:
See spec:
https://github.com/orbs-network/orbs-spec/blob/master/interfaces/primitives/protocol.proto#L26

https://golang.org/pkg/time/#Time.Unix

you need UnixNano()